plugin/src/templates/edge/ipx.ts

```diff
@@ -1,14 +1,12 @@
-import { Accepts } from "https://deno.land/x/accepts@2.1.1/mod.ts";
-import type { Context } from "netlify:edge";
+import { Accepts } from 'https://deno.land/x/accepts@2.1.1/mod.ts'
```

Comment:
These changes are because we've enabled prettier for Deno files because I can't work out how to make `deno fmt` run on a subsection of project files.

Reply:
It looks like you can do the following in deno.json. See https://deno.land/manual/getting_started/configuration_file#configuration-file

```
{
  "fmt": {
    "files": {
      "include": [
        "src/templates/edge"
      ]
    }
  }
}
```

Reply:
Thanks! I'm not sure if that will work with VS Code format-on-save, but I'll give it a go.

Reply:
This didn't seem to work, because there's no way to tell VS Code to try both